### PR TITLE
CIV-7277 Create service hearing model

### PR DESCRIPTION
**Pull request by adam-thomas-HMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-7277

### Change description ###
Added files for the service hearing model 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/model/hearingvalues/ServiceHearingValuesModel.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+@AllArgsConstructor
+public class ServiceHearingValuesModel {
+
+    private String hmctsServiceID;
+    private String hmctsInternalCaseName;
+    private String publicCaseName;
+    private boolean caseAdditionalSecurityFlag;
+    private List<CaseCategoryModel> caseCategories;
+    private String caseDeepLink;
+    @JsonProperty("caserestrictedFlag")
```
Comment (HarryH96) [inline src/main/java/uk/gov/hmcts/reform/civil/model/hearingvalues/ServiceHearingValuesModel.java:25]: What is this JsonProperty used for?

**Review by Gareth40343 — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/model/hearingvalues/ServiceHearingValuesModel.java (hunk 1 above)
Comment (adam-thomas-HMCTS) [reply src/main/java/uk/gov/hmcts/reform/civil/model/hearingvalues/ServiceHearingValuesModel.java:25]: That was used to map the caseRestricedFlag field to the correct form for ExUI, as I think that expects "caserestrictedFlag"

Comment (adam-thomas-HMCTS) [reply src/main/java/uk/gov/hmcts/reform/civil/model/hearingvalues/ServiceHearingValuesModel.java]: JSON annotation has now been removed

Comment (sankaviv1) [reply src/main/java/uk/gov/hmcts/reform/civil/model/hearingvalues/ServiceHearingValuesModel.java]: we need this json annotation as the field that exui is expecting is called caserestrictedFlag however in java we have to camel case the field name. so i think this should be reverted @adam-thomas-HMCTS @HarryH96

**Review by sankaviv1 — APPROVED** [review]

**Review by Gareth40343 — APPROVED** [review]

**Review by HarryH96 — APPROVED** [review]

**Pull request merged**